### PR TITLE
Link to Knowledge Sharing Sessions

### PR DESCRIPTION
**Pull request by jafeltra**
**Description:** This PR adds a section to the README that links to the Knowledge Sharing Sessions that would be helpful to watch before starting development.

**Testing Instructions:** Make sure the new section makes sense and is where you'd expect it.

**Related Issue:** N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,6 +181,11 @@ jest.mock('fhir-package-loader', () => {
 
 # For Developers
 
+## Intro to FHIR Package Loader
+
+To learn more about FHIR Package Loader, watch the Knowledge Sharing Sessions for [Developing FSH Tools](https://vimeo.com/990594107/c35c3f5e43) (view the slides [here](https://confluence.hl7.org/display/FHIR/FSH+Knowledge+Sharing+Sessions?preview=/256509612/256514908/KSS%203%20-%20Developing%20FSH%20Tools.pdf)) and [Developing FHIR Package Loader](https://vimeo.com/1008502493/76aba15913) (view the slides [here](https://confluence.hl7.org/display/FHIR/FSH+Knowledge+Sharing+Sessions?preview=/256509612/265096146/KSS%208%20-%20Developing%20FHIR%20Package%20Loader.pdf)).
```
Comment (cmoesel) [inline README.md]: As noted in similar PRs, the Developing FSH Tools video link needs to be fixed:
```suggestion
To learn more about FHIR Package Loader, watch the Knowledge Sharing Sessions for [Developing FSH Tools](https://vimeo.com/990594228/056b5c075f) (view the slides [here](https://confluence.hl7.org/display/FHIR/FSH+Knowledge+Sharing+Sessions?preview=/256509612/256514908/KSS%203%20-%20Developing%20FSH%20Tools.pdf)) and [Developing FHIR Package Loader](https://vimeo.com/1008502493/76aba15913) (view the slides [here](https://confluence.hl7.org/display/FHIR/FSH+Knowledge+Sharing+Sessions?preview=/256509612/265096146/KSS%208%20-%20Developing%20FHIR%20Package%20Loader.pdf)).
```

**Review by cmoesel — CHANGES_REQUESTED** [review]
Everything looks good, with the exception of the incorrect Developing FSH Tools video link. Once that's fixed, I think this is good to go!

File: README.md (hunk 1 above)
Comment (jafeltra) [reply README.md]: Fixed in [a9cfafd](https://github.com/FHIR/fhir-package-loader/pull/38/commits/a9cfafdfd8209e900cf08a318940df8106150a1f).

**Review by cmoesel — APPROVED** [review]
Looks good!

**Pull request merged**